.github/workflows/ci.yml

```diff
@@ -46,30 +62,23 @@ jobs:
 
       matrix:
         os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        python-version: ["3.10", "3.11", "3.12"]
+        environment: [ci-py310, ci-py313]
 
     steps:
       - name: Clone repository
         uses: actions/checkout@v4
-
-      - name: Setup environment
-        uses: actions/setup-python@v5
         with:
-          python-version: "${{ matrix.python-version }}"
-          cache: "pip"
-
-      - name: Install dependencies
-        run: |
-          python -m pip install -r ci/requirements/environment.txt
+          fetch-depth: 0
 
-      - name: Install marray
-        run: |
-          python -m pip install --no-deps -e .
+      - uses: prefix-dev/setup-pixi@v0.8.1
+        with:
+          pixi-version: v0.39.4
```

Comment:
is there a way to automatically update this (like, using `dependabot` or something similar)? Otherwise I think we will only upgrade whenever one of us remembers to manually do so, which will be pretty infrequent.

Reply:
I'm not sure, let me ask